pom.xml

```diff
@@ -79,7 +84,8 @@
 		<dependency>
 			<groupId>net.sf.jasperreports</groupId>
 			<artifactId>jasperreports</artifactId>
-			<version>6.1.1</version>
+			<version>4.6.0</version>
```

Comment:
If all versions are configurable, this should remain at version 6. Otherwise people using version 6 run into trouble. The reason that I didn't support multiple versions before is exactly this problem: you need the matching jasper version in the plugin as well. See #25